services/app-api/src/local-server.ts

```diff
@@ -176,15 +176,8 @@ router.get('/health_check', async (req: Request, res: Response) => {
 // OAuth token
 router.post('/oauth/token', async (req: Request, res: Response) => {
     const event = toLambdaEvent(req)
-    const context = createContext()
-    await handleLambdaResponse(
-        res,
-        oauthTokenHandler(
-            event,
-            context,
-            () => {}
-        ) as Promise<APIGatewayProxyResult>
-    )
+    // What is the expected behavior here / is there a better way to do this?
+    await handleLambdaResponse(res, oauthTokenHandler(event))
 })
```

Comment:
Might be worth creating a ticket for this. I was planning on dismissing as won't fix since there hasn't been a change here.